docs/manual/compose-server.md

```diff
@@ -59,7 +70,7 @@ If you're doing this multiple times, it's strongly recommended to create a cache
 directory:
 
 ```
-# rpm-ostree compose tree --cachedir=cache --repo=/srv/centos-atomic/build-repo sig-atomic-buildscripts/centos-atomic-host.json
+# rpm-ostree compose tree --unified-core --cachedir=cache --repo=/srv/centos-atomic/build-repo sig-atomic-buildscripts/centos-atomic-host.json
```

Comment:
Aside: hmm, we should probably update these examples to use FCOS, even if one wouldn't normally run rpm-ostree directly to build there.

Reply:
Probably best to refer to actual working pipelines like https://github.com/CentOS/sig-atomic-buildscripts - will look at some of that potentially as a followup.